step-controller/step-controller-server/src/main/java/step/core/deployment/AbstractStepServices.java

```diff
@@ -96,4 +95,19 @@ protected void checkRightsOnBehalfOf(String right, String userOnBehalfOf) {
 			throw new AuthorizationException(ex.getMessage());
 		}
 	}
+
+	/**
+	 * The ObjectHookInterceptor.aroundReadFrom can only be used for POST request passing an entity as request BODY
+	 * This method can be used as helper for all other cases where checking if the entity is acceptable in given context (i.e. DELETE request...(
+	 * @param entity the entity to be asserted
+	 */
+	protected void assertEntityIsAcceptableInContext(AbstractIdentifiableObject entity) {

```

Comment:
I would maybe change the name of this helper method to something more meaningful like: isObjectEditableInContext